File: Problem/BOJ_5597_과제 안 내신 분/src/Main.java
import java.util.*;
import java.io.*;


public class Main {
    public static void main(String[] args) throws IOException {
        System.setIn(new FileInputStream("src\\input.txt"));
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        HashSet<Integer> students = new HashSet<>();
        for (int std = 1; std <= 30; std++) {
            students.add(std);
        }

        for (int i = 0; i < 28; i++) {
            students.remove(Integer.parseInt(br.readLine()));
        }

        ArrayList<Integer> studentList = new ArrayList<>(students);
        Collections.sort(studentList);

        for (int student : studentList) {
            System.out.println(student);
        }
    }
}

File: Problem/BOJ_5597_과제 안 내신 분/main.py
from sys import stdin


def main():
    stdin = open("./input.txt", "r")
    students = set([i for i in range(1, 31)])

    for _ in range(28):
        student = int(stdin.readline())
        students.discard(student)

    array_students = list(students)
    array_students.sort()
    print(array_students[0])
    print(array_students[1])


if __name__ == '__main__':
    main()